flashcard_runner.rb

```diff
@@ -1,13 +1,18 @@
-require './lib/card'
+# require './lib/card'
 require './lib/deck'
 require './lib/round'
+require './lib/card_generator'
 
```

Comment:
I am keeping the commented out code in for the time being just to see the difference between the old and the new method of loading cards using the Generator.